Technical-Steering-Committee/principles.md

```diff
@@ -0,0 +1,47 @@
+
+## Zowe does not compromise on mainframe security standards
+**Principle:** Security is paramount
+
+We commit to secure development practices. We'll follow/comply/adhere to/obey with /conform to cybersecurity regulations requirements. We will strive to never introduce a feature or change that circumvents or allows circumvention of z/OS security infrastructure. We will regularly/continuously perform vulnerability testing of the Zowe project source code and any used third party libraries. If vulnerabilities are found they will be resolved with fix or workaround will be provided.
+
+## Zowe provides modern interfaces to access and exploit mainframe
+**Principle:** Modern and effective interfaces
+
+We will provide the foundations and tools for all the vendors existing within the ecosystem to easily interface to and develop more interfaces to z/OS. The tools should be simple, effective, actively used and developed. The users should be able to leverage the provided tools and interfaces to get more value out of their mainframe environments. Examples domains for such tools and interfaces include web, mobile, APIs (C interfaces to HLASM, other programming interfaces, REST APIs, ...) and modern Developer Environments.
+
+## Collaboration between mainframe and other platforms - both mainframe and distributed - benefits both sides
+**Principle:** Hybrid
+
+We will bring lessons from the non-mainframe world to the mainframe platform as well as providing the opportunity for the non-mainframe world to benefit from the knowledge gained in the mainframe world in order to build secure and performant hybrid environments in which mainframe plays core role. To do so we intend to lower the entry barrier to the mainframe environment by leveraging the technologies famililar to modern developers and supporting mainframe education.
+
+## We will listen to our stakeholders and elicit feedback from them.
+**Principle**: Stakeholder-centric
+
+We are stakeholder-centric project where stakeholders include end-users, exploiters, deploying enterprises or the Zowe Advisory Council. We will keep the best interests of our stakeholders in mind and elicit feedback from them as much as possible, incorporate feedback when planning future work, and keep our stakeholders informed on Zowe's evolution and how their feedback is received and responded to.
+
+## The Zowe does not depend on and is not owned by any specific Vendor
+**Principle:** Vendor Neutral
+
+We will keep Zowe vendor neutral. No company can claim that it owns Zowe. If we use technology owned by a vendor or company (e.g. z/OS) we need a very strong argument and a means to move away from that technology if the vendor intends to leverage the technology counter to our principles. Such technology also needs to be accessible to anyone wanting to participate and contribute to the Zowe project. 
```

Comment:
*we need a very strong argument and a means to move away from that

I think this is fairly strongly worded. Sometimes, alternatives do not exist: only 1 way of implementation exists and including things like "and a means to move away from that" may not be always possible. My recommendation is to add "we need a very strong argument and if possible, a means to move away from that"

Reply:
As this change would be material, I can't do the change without doing the vote from the start. Do you feel that the change in wording you propose is important enough?

I believe that the general idea is that we can use the technologies e.g. Slack we use and it was discussed that the principle is fine with it. 

And as z/OS is mentioned in the paragraph we have no means away from z/OS so I think that the idea is already to some extent present there.